autogpt_platform/frontend/src/components/CustomNode.tsx

```diff
@@ -104,6 +107,7 @@ export function CustomNode({
   >();
   const isInitialSetup = useRef(true);
   const flowContext = useContext(FlowContext);
+  const api = useBackendAPI();
```

Comment:
This is how I want all API usage to look eventually. I just didn't want to do all the refactoring in this PR, so that will come later.